impala/_thrift_api.py

```diff
@@ -122,9 +127,20 @@ def __init__(self, uri_or_host, port=None, path=None, cafile=None, cert_file=Non
       self.proxy_auth = self.basic_proxy_auth_header(parsed)
     else:
       self.realhost = self.realport = self.proxy_auth = None
-    self.__auth_cookie_names = auth_cookie_names
-    self.__auth_cookie = None
-    self.__auth_cookie_expiry = None
+    if not http_cookie_names:
+      # 'http_cookie_names' is explicitly set as an empty value ([], or '').
```

Comment:
nit: This comment is a bit misleading as the if condition is True even if 'http_cookie_names' is not set explicitly when creating an ImpalaHttpClient() object. Maybe something like his would be  more straightforward:
If 'http_cookie_names' was explicitly set to an empty value ([] or '') in connect().

Reply:
Fixed.